package.json

```
@@ -5,24 +5,41 @@
   "main": "index.js",
   "scripts": {
     "test": "eslint *.js"
+    "format": "prettier --write '**/*.{js,md,css,scss,yaml,yml,json}'",
+    "lint": "eslint ./ --max-warnings 0 && prettier --write '**/*.{js,md,css,scss,yaml,yml,json}'",
+    "preflight": "npm run lint && npm run test",
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/wagtail/eslint-config-wagtail.git"
   },
+  "dependencies": {
+    "eslint-config-airbnb": "^9.0.1",
+  },
+  "devDependencies": {
+    "eslint": "^2.12.0",
+    "jest": "24.7.1",
```

Why pin to a specific version?

```suggestion
    "jest": "^24.7.1",
```